File: app/src/main/java/com/annapol04/munchkin/engine/Curse.java
package com.annapol04.munchkin.engine;

import android.support.annotation.DrawableRes;
import android.support.annotation.StringRes;

public class Curse extends Card {

    public Curse (@StringRes int name, @StringRes int description, @DrawableRes int imageResourceID) {
        super(name, description, imageResourceID);
    }

}

File: app/src/main/java/com/annapol04/munchkin/engine/Curse.kt
package com.annapol04.munchkin.engine

import android.support.annotation.DrawableRes
import android.support.annotation.StringRes

class Curse(@StringRes name: Int,
            @StringRes description: Int,
            @DrawableRes imageResourceID: Int)
    : Card(name, description, imageResourceID)
